components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -322,6 +324,23 @@ class ExperimentsTest {
         assertEquals("second-id", experiments.activeExperiment!!.experiment.id)
     }
 
+    @Test
+    fun `experiment enrollment recorded in Glean`() {
```

Comment:
Since we're doing this, I'd also add a test for making sure we correctly unenroll.

Reply:
This test actually does this, in the call to `onExperimentsUpdated` where the empty list is passed in, unenrollment occurs and then the test does the `assertFalse(Glean.testIsExperimentActive("second-id"))` to verify.  I admit that the name of the test was misleading so I renamed the test to better indicate that it tests both enrollment and unenrollment.

Reply:
Ok, let's drop a comment before that line to call that out.

Reply:
Already assumed you would ask so I commented the whole test to explain its steps :)

Reply:
Oh my, now you're scaring me ;) Great job!